public/chat/santaChat.js

```diff
@@ -59,13 +61,11 @@ $(async function() {
           $(chatElement).find('[data-name="chatDate"]').text('Just now...');
           $(chatElement).find('[data-name="chatFrom"]').text('From: you');
 
-          $(chatElement).find('button').on('click', function() {
-            $.post(`${apiUrl}/delete`, { _id: response.insertedId }, result => {
-              if (result.success) $(chatElement).remove();
-              showAlert(result);
-            });
+          $(chatElement).find('[data-name="msgDelete"]').on('click', function() {
+            $('#deletePairDialog').prop('hidden', false);
```

Comment:
here as well, what's this?

Reply:
I'm not sure, I'd have to think about it